libbeat/docs/http-endpoint.asciidoc

```diff
@@ -0,0 +1,149 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/http-endpoint.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[http-endpoint]]
+== HTTP Endpoint
```

Comment:
I think it's fine to include this content here since it's experimental, but at some point, we probably do what a separate section called something like Monitoring APIs. I don't think "http endpoint" is very descriptive of what these config options do.